src/op_mode/ping.py

```diff
@@ -115,6 +115,10 @@
         'type': '<bytes>',
         'help': 'Number of bytes to send'
     },
+    'source-address': {
+        'ping': '{command} -I {value}',
+        'type': '<X.X.X.X> <h:h:h:h:h:h:h:h>',
```

Comment:
Can we use lowercase x.x.x.x as in other places?

Reply:
Good point.